src/stirling/source_connectors/perf_profiler/perf_profiler_bpf_test.cc

```diff
@@ -210,37 +209,37 @@ class ContainerSubProcesses final : public PerfProfilerTestSubProcesses {
   std::vector<std::unique_ptr<ContainerRunner>> sub_processes_;
 };
 
+template <bool FastTest>
```

Comment:
Could you explain fast tests vs slow tests? I don't really understand what we're trying to achieve with the distinction.

Reply:
The "slow" test accepts all the default profiler parameters, but with the 30 sec. data sampling periodicity, the test inherently runs slower. The "fast" test overrides the defaults, but the test case will complete in less time. At some point in the history, we had (or were going toward) all tests using the "fast" methodology but a reviewer suggested that we keep at least one test with the slow/default setup. This PR does not change the existing setup, but it does provide a nicer way to identify which test is in which category by naming the templated test fixtures accordingly.